### PR TITLE
Adapt the client brownfield API commands (issue #64)

### LINKED ISSUE #64: Adapt the client brownfield API commands
It's required to implement new workflow for Brownfield importation activities using API V3.

Adding support for:
- servers
- vpcs
- floating_ips
- volumes
- policies

New _Brownfield_ commands:
**cio brownfield cloud-accounts \<CMD>**:

- **list**                    -> Lists the cloud accounts that support importing resources
	GET	/v3/brownfield/cloud_accounts

**cio brownfield import-\<resources>**:

- **import-servers**        -> Import servers candidates, by given cloud account id
	PUT	/v3/brownfield/cloud_accounts/:cloud_account_id/import_servers

- **import-vpcs**           -> Import VPCs candidates, by given cloud account id
	PUT	/v3/brownfield/cloud_accounts/:cloud_account_id/import_vpcs

- **import-floating-ips**   -> Import Floating IPs candidates, by given cloud account id
	PUT	/v3/brownfield/cloud_accounts/:cloud_account_id/import_floating_ips

- **import-volumes**        -> Import volumes candidates, by given cloud account id
	PUT	/v3/brownfield/cloud_accounts/:cloud_account_id/import_volumes

- **import-policies**        -> Import policies candidates, by given cloud account id
	PUT	/v3/brownfield/cloud_accounts/:cloud_account_id/import_policies

# Expected Behavior

# Current Behavior

# Possible Solution

# Steps to Reproduce (for bugs)

    1.
    2.
    3.
    4.

# Context and environment

### PR DESCRIPTION
**Pull request by pcantera**
# Description

It's required to implement new workflow for Brownfield importation activities using API V3.

Adding support for:
- servers
- vpcs
- floating_ips
- volumes
- policies

New _Brownfield_ commands:
imagen

**cio brownfield cloud-accounts \<CMD>**:

- **list**                    -> Lists the cloud accounts that support importing resources
	GET	/v3/brownfield/cloud_accounts

**cio brownfield import-\<resources>**:

- **import-servers**        -> Import servers candidates, by given cloud account id
	PUT	/v3/brownfield/cloud_accounts/:cloud_account_id/import_servers

- **import-vpcs**           -> Import VPCs candidates, by given cloud account id
	PUT	/v3/brownfield/cloud_accounts/:cloud_account_id/import_vpcs

- **import-floating-ips**   -> Import Floating IPs candidates, by given cloud account id
	PUT	/v3/brownfield/cloud_accounts/:cloud_account_id/import_floating_ips

- **import-volumes**        -> Import volumes candidates, by given cloud account id
	PUT	/v3/brownfield/cloud_accounts/:cloud_account_id/import_volumes

- **import-policies**        -> Import policies candidates, by given cloud account id
	PUT	/v3/brownfield/cloud_accounts/:cloud_account_id/import_policies

# Related Issue
Closes #64

# Motivation and Context
- Brownfield commands adaptation.
- API V3 should be used.

# How Has This Been Tested?

# Screenshots (if appropriate)

# Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

# Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by lttito — COMMENTED** [review]
Don't we have a changelog in CIO? I think it's a good idea to include one if we don't have one yet...

**Comment by pcantera** [comment]
There isn't a dedicated file; and since code repository is public accessible it's shown at releases page. Please take a look on this url: https://github.com/ingrammicro/cio/releases

**Review by lttito — APPROVED** [review]
:+1:

File: cmd/import_candidates_cmd.go [hunk 1]
```diff
@@ -29,103 +31,162 @@ func WireUpImportCandidate(c *cli.Context) (ds *clientbrownfield.ImportCandidate
...
+				log.Info("Done!")
+			}
+			return ca
+		}
+		time.Sleep(5 * time.Second)
+	}
+}
+
+// ImportCandidateServers subcommand function
+func ImportCandidateServers(c *cli.Context) error {
```
Comment (pbanos) [inline cmd/import_candidates_cmd.go:58]: We should not longer talk about candidates: the action now is ImportServers. This applies to other functions below and to the name of the file.

File: clientbrownfield/subcommands.go [hunk 2]
```diff
@@ -11,74 +11,72 @@ func SubCommands() []cli.Command {
...
+					Name:  "id",
+					Usage: "Cloud account Id",
 				},
 			},
 		},
 		{
-			Name:   "import-volume",
-			Usage:  "Import volume import candidate, given its id",
-			Action: cmd.ImportCandidateVolumeImport,
+			Name:   "import-kb-clusters",
```
Comment (pbanos) [inline clientbrownfield/subcommands.go:62]: - If your are going to shorten the Kubernetes word, use the widespread abbreviation `k8s`
- We should not add a kubernetes cluster import command until the CLI manages kubernetes clusters

**Review by pbanos — CHANGES_REQUESTED** [review]

File: clientbrownfield/subcommands.go (hunk 2 above)
Comment (pcantera) [reply clientbrownfield/subcommands.go:62]: understood, kubernetes command is removed from current context; hiding command until kubernetes management is implemented.

**Review by lttito — APPROVED** [review]

**Review by pbanos — APPROVED** [review]
:+1:

**Review by dcd000 — APPROVED** [review]

**Review by spatino — APPROVED** [review]

**Review by chris-m-h — APPROVED** [review]

**Pull request merged**